### DISCUSSION TITLE
NPM unix sock

### DISCUSSION THREAD

**Discussion by corgitolecorgi** [NginxProxyManager/nginx-proxy-manager#3314] [Q&A]
Hello :) 

I'm trying to use NPM with a Django application, but there is a problem.
I create a linux service like this : 

**PlanlifeTest.socket**
``` shell
[Unit]
Description=Planlife TEST

[Socket]
ListenStream=/run/planlifetest.sock

[Install]
WantedBy=sockets.target
```

**PlanlifeTest.service**
``` shell
[Service]
User=gitlab-runner
Group=www-data
WorkingDirectory=/home/gitlab-runner/test
ExecStart=/home/gitlab-runner/test/venv/bin/gunicorn \
          --access-logfile - \
          --workers 3 \
          --bind unix:/run/planlifetest.sock \        
          planlife.wsgi:application

[Install]
WantedBy=multi-user.target
```

**And I try to configure it on NPM :** 

![image](https://github.com/NginxProxyManager/nginx-proxy-manager/assets/35119296/3912d247-d1fa-4949-835e-5fa6a32d88ee)
![image](https://github.com/NginxProxyManager/nginx-proxy-manager/assets/35119296/80425732-a0e2-4913-b100-1f9fb0aa6baa)
![image](https://github.com/NginxProxyManager/nginx-proxy-manager/assets/35119296/e8886727-2c64-4f4d-bc93-c935030dcaaa)
![image](https://github.com/NginxProxyManager/nginx-proxy-manager/assets/35119296/5c4731af-77f2-456f-8cfa-c8237b11c364)

But as you imagine it's not working :'( i'm getting a 502 error, someone may help me ? :)

**Comment by jeliastam** [comment]
hopefully you figured it out, but I landed here trying to use unix sockets with NPM and used your example to make mine work... so all I can tell you if that you did it right and the problem must be downstream from NPM.
